grails-app/controllers/org/pih/warehouse/UrlMappings.groovy

```diff
@@ -967,6 +967,16 @@ class UrlMappings {
             action = [POST: "submitCount"]
         }
 
+        "/api/facilities/$facility/cycle-counts/items/$cycleCountItemId" {
```

Comment:
I was going to say that we should do `/api/facilities/$facility/cycle-counts/$cycleCountId/items/$cycleCountItemId` just to be extra clear and consistent with the other API but ultimately I think this is fine since item id is already enough to determine uniqueness.